### DISCUSSION TITLE
Class Names in Triggers

### DISCUSSION THREAD

**Discussion by princealexi25** [smasherprog/EqTool#169] [Ideas]
would be a great feature if I do a /w in zoone and a name match is found in current trigger to add class names to char names to list so I know what buffs to cast without /whotarget

**Comment by smasherprog** [comment]
Great idea.. adding that now for the next huge release

**Comment by smasherprog** [comment]
Added for this upcoming release 
![image](https://github.com/user-attachments/assets/d65efc2d-79ea-422b-b413-ccd92856010b)

**Comment by smasherprog** [comment]
https://discord.gg/Z6RPUMNcY2
Discord is a good place to post more ideas!

**Comment by princealexi25** [comment]
I'm a coder, just makes sense to place in repo.  disco seems superfluous.
